.pre-commit-config.yaml

```diff
@@ -33,6 +33,7 @@ repos:
     hooks:
       - id: debug-statements
       - id: check-docstring-first
+      - id: trailing-whitespace
```

Comment:
This helps 70% of my issues with the markdown pre-commit check further down, thus fairly time saving.

Reply:
Turns out this interfere with the snapshot tests.